app/controllers/account/names_controller.rb

```diff
@@ -25,5 +25,9 @@ def account_name_form_params(user)
     def redirect_if_name_exists
       redirect_to root_path if current_user.name.present?
     end
+
+    def next_path
```

Comment:
don't know if repeating this method is better than putting it in application controller. It's hard to find somewhere else to stick it.